CMakeLists.txt

```diff
@@ -555,6 +555,11 @@ else()
         -Wunused-result
         -Wwrite-strings)
 
+    if(ENABLE_MAC)
+        # GNU extensions are supported and good practice in Objective-C
+        list(APPEND WARNING_CANDIDATES -Wno-gnu)
```

Comment:
This disables warnings when building on Mac, not when building Obj-C code.

Consider using `add_compile_options(-Wno-gnu)` in macosx/CMakeLists.txt instead. I'd rather not use extensions though.

Reply:
Thanks, I was searching how to do that.